packages/platform/atoms/calendar-settings/wrappers/CalendarSettingsPlatformWrapper.tsx

```diff
@@ -0,0 +1,240 @@
+                                    type={connectedCalendar.integration.type}
+                                    isChecked={cal.isSelected}
+                                    destination={cal.externalId === destinationCalendarId}
+                                    credentialId={cal.credentialId}
+                                  />
+                                );
+                              })}
+                            </ul>
+                          </div>
+                        </AppListCard>
+                      );
+                    }
+                    return (
+                      <Alert
+                        key={`alert-${connectedCalendar.credentialId}`}
+                        severity="warning"
+                        title={t("something_went_wrong")}
+                        message={<span>{t("calendar_error")}</span>}
+                        iconClassName="h-10 w-10 ml-2 mr-1 mt-0.5"
+                        actions={
+                          <div className="flex w-32 justify-end">
+                            <PlatformDisconnectIntegration
+                              credentialId={connectedCalendar.credentialId}
+                              trashIcon
+                              buttonProps={{ className: "border border-default" }}
+                              slug={connectedCalendar.integration.slug}
+                            />
+                          </div>
+                        }
+                      />
+                    );
+                  })}
+                </List>
+              </CalendarSettings>
+            );
+          }}
+        />
+      </div>
+    </AtomsWrapper>
+  );
+};
+
+const CalendarSettingsHeading = () => {
+  const { t } = useLocale();
+
+  return (
+    <div className="border-subtle border-b p-6">
+      <div className="flex items-center justify-between">
+        <div>
+          <h4 className="text-emphasis text-base font-semibold leading-5">{t("check_for_conflicts")}</h4>
+          <p className="text-default text-sm leading-tight">{t("select_calendars")}</p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const PlatformDisconnectIntegration = (props: {
+  credentialId: number;
+  label?: string;
+  slug?: string;
+  trashIcon?: boolean;
+  isGlobal?: boolean;
+  onSuccess?: () => void;
+  buttonProps?: ButtonProps;
+}) => {
+  const { t } = useLocale();
+  const { onSuccess, credentialId, slug } = props;
+  const [modalOpen, setModalOpen] = useState(false);
+  const { toast } = useToast();
+  const { mutate: deleteCalendarCredentials } = useDeleteCalendarCredentials({
+    onSuccess: () => {
+      toast({
+        description: t("app_removed_successfully"),
+      });
+      setModalOpen(false);
+      onSuccess && onSuccess();
+    },
+    onError: () => {
+      toast({
+        description: t("error_removing_app"),
+      });
+      setModalOpen(false);
+    },
+  });
+
+  return (
+    <DisconnectIntegrationComponent
```

Comment:
In prod web app disconnect integration looks like:
<img width="656" alt="Screenshot 2024-08-12 at 10 14 55" src="https://github.com/user-attachments/assets/11dcf621-f334-4109-a7c1-1732a69a5ca9">

but in atoms the padding looks very big in examples app:
<img width="669" alt="Screenshot 2024-08-12 at 10 16 50" src="https://github.com/user-attachments/assets/7cd9f242-d15a-4319-aabe-3589a1cfde02">

So it would be good to add className to `DisconnectIntegrationComponent` and in web we pass existing padding and in atoms we pass smaller padding also given that in atoms the warning icon is not displayed so we need less space.

Reply:
@supalarry I think the padding looks very big because the danger icon that you see in the web app dialog is missing  and thats what causes it. Otherwise both the dialogs have the same styles, can confirm that.